<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushEditDialog</class>
 <widget class="QDialog" name="BrushEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brush editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="propertiesWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="styleLabel">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="styleComboBox">
          <item>
           <property name="text">
            <string>NoBrush</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SolidPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dense1Pattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dense2Pattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dense3Pattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dense4Pattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dense5Pattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dense6Pattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dense7Pattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HorPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VerPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CrossPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BDiagPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FDiagPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DiagCrossPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LinearGradientPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RadialGradientPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ConicalGradientPattern</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TexturePattern</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="colorWidget" native="true">
          <layout class="QHBoxLayout" name="colorLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="colorLabel">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="gradientWidget" native="true">
          <layout class="QVBoxLayout" name="gradientLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="gradientLabel">
             <property name="text">
              <string>Gradient:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="textureWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="textureIconLabel">
             <property name="text">
              <string>icon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="textureLabel">
             <property name="text">
              <string>Texture</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="textureButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="transformWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="transformLabel">
             <property name="text">
              <string>Transform</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="transformEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="transformButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
